contrib/examples/walkthrough/ups-instance.yaml

```diff
@@ -7,6 +7,7 @@ spec:
   serviceClassName: user-provided-service
   planName: default
   parameters:
-    credentials:
-      name: root
-      password: letmein
+    inline:
+      credentials:
+        name: root
+        password: letmein
```

Comment:
yes I would much prefer something like this. Moving the union up a level allows for the simple case to remain simple - meaning the case of just passing in a set of name/value pairs would be done by specifying them as children of "parameters" w/o an "inline" wrapper.

As we discussed off-line, what do people think of something like this (keep in mind this shows/covers all possible scenarios):   https://gist.github.com/duglin/1422e0eafc7eca22157af96a3f4a84d0